### PR TITLE
1878: Improve argument check in SKARA CLI

### PR DESCRIPTION
**Pull request by zhaosongzs**
Currently, if a user misuses a command, in some cases, SKARA CLI crashes and throws a stack trace, which is bad and does not provide useful information to the user. 

After investigating, I found that in many places, we don't check whether an option is followed by an argument, and it would trigger some issues.

In this patch, the program would always check the presence of an argument  before attempting to retrieve it for an option.

---------
### Progress
- [x] Change must be properly reviewed (1 review required, with at least 1 [Reviewer](https://openjdk.org/bylaws#reviewer))
- [x] Change must not contain extraneous whitespace

### Issue
 * [SKARA-1878](https://bugs.openjdk.org/browse/SKARA-1878): Improve argument check in SKARA CLI

### Reviewers
 * [Erik Joelsson](https://openjdk.org/census#erikj) (@erikj79 - **Reviewer**)

### Reviewing
<details><summary>Using <code>git</code></summary>

Checkout this PR locally: \
`$ git fetch https://git.openjdk.org/skara.git pull/1502/head:pull/1502` \
`$ git checkout pull/1502`

Update a local copy of the PR: \
`$ git checkout pull/1502` \
`$ git pull https://git.openjdk.org/skara.git pull/1502/head`

</details>
<details><summary>Using Skara CLI tools</summary>

Checkout this PR locally: \
`$ git pr checkout 1502`

View PR using the GUI difftool: \
`$ git pr show -t 1502`

</details>
<details><summary>Using diff file</summary>

Download this PR as a diff file: \
<a href="https://git.openjdk.org/skara/pull/1502.diff">https://git.openjdk.org/skara/pull/1502.diff</a>

</details>

### Webrev
[Link to Webrev Comment](https://git.openjdk.org/skara/pull/1502#issuecomment-1507734950)

### CODE REVIEW

File: cli/src/main/java/org/openjdk/skara/cli/GitSync.java [hunk 1]
```diff
@@ -330,29 +330,18 @@ public void sync() throws IOException, InterruptedException {
...
                 branches.add(branch.trim());
             }
-        } else {
-            var lines = repo.config("sync.branches");
-            if (lines.size() == 1) {
-                var requested = lines.get(0).split(",");
-                for (var branch : requested) {
-                    branches.add(branch.trim());
-                }
-            }
```
Comment (erikj79) [inline cli/src/main/java/org/openjdk/skara/cli/GitSync.java:345]: What happened to this part?

**Review by erikj79 — COMMENTED** [review]
Good fix overall. I see you updated copyright years in some files, please make sure all the files you have changed are updated.

File: cli/src/main/java/org/openjdk/skara/cli/GitSync.java (hunk 1 above)
Comment (erikj79) [reply cli/src/main/java/org/openjdk/skara/cli/GitSync.java:345]: Oh, right, I see now.

**Review by erikj79 — APPROVED** [review]
Looks good!

**Comment by zhaosongzs** [comment]
Thanks for the review!
/integrate

**Pull request closed without merging**